assets/src/application/ui/display/espressoIcons/EspressoIcon.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { svgPath } from './svgPaths';
+import { EspressoIconProps } from './types';
+
+/**
+ * Custom Event Espresso Dashicons
+ */
+const EspressoIcon = React.forwardRef<SVGSVGElement, EspressoIconProps>(
+	({ name, svgSize = 20, className, ...otherProps }, ref) => {
+		const path = svgPath[name];
+
+		if (!path) {
+			return null;
+		}
+
+		const htmlClass = classNames(className, 'dashicon', `dashicons-${name}`, 'espresso-icon');
```

Comment:
`className`

Reply:
oh, I see why this was used this way, what about:

```
		const className = classNames(otherProps.className, 'dashicon', `dashicons-${name}`, 'espresso-icon');
```
and `className` can be removed from the destructurization from the above

Reply:
Done